src/fhir.js

```diff
@@ -446,9 +446,10 @@ function toCode(f) {
       return codings.length === 1 ? codings[0] : codings;
     }
   } else if (typeName === 'Coding') {
-    return new cql.Code(f.code.value, f.system ? f.system.value : f.system, f.version ? f.version.value : f.version, f.display ? f.display.value : f.display);
+    return new cql.Code(f.code ? f.code.value : f.code, f.system ? f.system.value : f.system, f.version ? f.version.value : f.version, f.display ? f.display.value : f.display);
+  } else if (typeName === 'code') {
+    return f.value;
   }
-  return new cql.Code(f.code.value);
```

Comment:
If I am reading this correctly, the old version had this as a catch-all in case none of the `if` / `else if` conditions were met.  Now `toCode` could potentially return nothing; are we concerned about that?

Reply:
You are reading that correctly.  Before, it assumed that if it's not a `CodeableConcept`, and it's not a `Coding`, then it _must_ be a `code`!  But that was an invalid assumption.

In the cases where it is not a code, it is appropriate to return `undefined`.  Think of it like this -- the caller asked for a code, and there was no code; so don't return anything (whereas before, we tried to return something else wearing a code costume!).

Reply:
That makes sense; hence the test you added below.